inc/Addon/Varnish/Subscriber.php

```diff
@@ -116,4 +117,36 @@ public function clean_home( $root, $lang ) {
 		$this->varnish->purge( $home_url );
 		$this->varnish->purge( $home_pagination_url );
 	}
+
+	/**
+	 * Clears URLs in Varnish cache.
+	 *
+	 * @param array $urls The path of home cache file.
+	 */
+	public function clean_urls( $urls ) {
+		if ( ! $this->should_purge() ) {
+			return;
+		}
+
+		foreach ( $urls as $data ) {
+			foreach ( $data['files'] as $file_path ) {
+				if ( strpos( $file_path, '#' ) ) {
+					// URL with query string.
+					continue;
+				} else {
```

Comment:
This `else` is not needed. Why? If the `if` conditional is true, the first `foreach` skips the rest of the code. Therefore, the `else` never fires.